File: test_practice/hello.java


public class hello { 

   public static void main(String[] args) { 
   	int[] a = {1 ,2 ,3 ,4, 2};
      System.out.println("Hello, World: " + max(a));
   }


   public static int max(int[] list){
   		int largest = list[0];
   		for(int i = 1; i < list.length; i++){
   			if(largest < list[i]){
   				largest = list[i];
   			}
   		}
   		return largest;
   }

}

File: test_practice/hello.cpp
#include <iostream>
#include <vector>
using namespace std;



void pt (const vector<string> &vec){
	std::vector<string>::const_iterator itor = vec.begin();
	std::vector<string>::const_iterator end = vec.end();
	for(; itor != end; ++itor){
		cout << *itor << endl;
	}
}

int main(){

	std::vector<string> array;
	for(int i = 0; i < 10; i++){
		array.push_back("pollo");
	}

	//std::cout << "Hello World!" << array[0] << "\n";
	pt(array);

}

